src/lib/routes/proxy-api/index.ts

```diff
@@ -156,6 +156,9 @@ export default class ProxyController extends Controller {
                 ProxyController.createContext(req),
             );
         }
+
+        res.set('Cache-control', 'public, max-age=2');
```

Comment:
this one gently asks the browser to cache for at least 2 seconds, even when the SDK is configured to poll more often.